deno/deepgram_audio_summary/src/mod.ts

```diff
@@ -0,0 +1,40 @@
+export default async function (req: any, res: any) {
+    const APIkey = req.variables["DEEPGRAM_API_KEY"];
+    let fileUrl = "";
+
+    try {
+        fileUrl = JSON.parse(req.payload);
+    } catch (error) {
+        return res.json("Invalid JSON string");
+    }
+
+    const response = await fetch(
+        "https://api.deepgram.com/v1/listen",
+        {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                Authorization: `Token ${APIkey}`,
+            },
+            body: JSON.stringify({ url: fileUrl }),
+        }
+    );
+
+    if (!APIkey || !fileUrl) {
+        return res.json("Please provide valid APIkey and fileUrl");
+    }
+
+    else if(response.status !== 201) {
+        const error = await response.json();
+        return res.json({
+            success: false,
+            message: `Encountered an error trying to make a request: ${error.reason}`,
```

Comment:
I used a correct API key but got this error:

```json
{
  "response": {
    "success": false,
    "message": "Encountered an error trying to make a request: undefined"
  },
  "stdout": "",
  "stderr": ""
}
```

Is it working for you? Also, `undefined` is not that helpful.